tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml

```diff
@@ -0,0 +1,12 @@
+---
+namespace.monitoring:
+  flows:
+    CreateAlertDashboards:
+      tags:
+        - tendrl/monitor
```

Comment:
How is this even going to work? This tag "tendrl/monitor" is for the node-agent on the tendrl server.  Change it to "tendrl/integrations/monitoring"

- Define tag "tendrl/integrations/monitoring" in monitoring-integration.conf.yaml , like this https://github.com/Tendrl/node-agent/blob/master/etc/tendrl/node-agent/node-agent.conf.yaml.sample#L13